container/containerd/client.go

```diff
@@ -125,6 +130,21 @@ func (c *client) Version(ctx context.Context) (string, error) {
 	return response.Version, nil
 }
 
+func (c *client) ContainerFsUsage(ctx context.Context, snapshotter, snapshotkey string) (*common.FsUsage, error) {
+	usage, err := c.snapshotsService.Usage(ctx, &snaptshotapi.UsageRequest{
+		Snapshotter: snapshotter,
+		Key:         snapshotkey,
+	})
+	if err != nil {
+		return nil, err
+	}
+	return &common.FsUsage{
+		BaseUsageBytes:  uint64(usage.Size_),
+		TotalUsageBytes: uint64(usage.Size_),
```

Comment:
is it correct that these are both set to `usage.Size`? 

BaseUsageBytes is "Number of bytes consumed by a container through its root filesystem." vs TotalusageBytes is "Total Number of bytes consumed by container." ?

Reply:
Yes ,for containerd ,there is no log file(used to store logs from container's stdout or stderr) (At least ,I didn't find it T_T) . But for docker and crio , there are log files, so when collect fs usage, the usage of log file is counted.

By the way, kubelet storge container's  logs in another place (`/var/log/pods`), and it is counted by kubelet when it caculate the usage. see `makeContainerStats` in `kubelet/stats/cri_stats_provider.go`

```go
// NOTE: This doesn't support the old pod log path, `/var/log/pods/UID`. For containers
	// using old log path, empty log stats are returned. This is fine, because we don't
	// officially support in-place upgrade anyway.
	var (
		containerLogPath = kuberuntime.BuildContainerLogsDirectory(meta.GetNamespace(),
			meta.GetName(), types.UID(meta.GetUid()), container.GetMetadata().GetName())
		err error
	)
	result.Logs, err = p.getPathFsStats(containerLogPath, rootFsInfo)
	if err != nil {
		klog.Errorf("Unable to fetch container log stats for path %s: %v ", containerLogPath, err)
	}
	return result
```

Reply:
Shouldn't cadvisor take the log size of `/var/log/pods` in account when counting `TotalusageBytes`?